.circleci/config.yml

```diff
@@ -720,66 +820,153 @@ workflows:
           name: Unit Test
           requires:
             - Build
-
       - integration-test-part:
-          name: Integration Test Part 1
+          name: Integration Test 1
           index: 0
-          total: 1
+          total: 3
           requires:
             - Build
       - integration-test-part:
-          name: Integration Test Part 2
+          name: Integration Test 2
           index: 1
-          total: 1
+          total: 3
+          requires:
+            - Build
+      - integration-test-part:
+          name: Integration Test 3
+          index: 2
+          total: 3
           requires:
             - Build
-
       - test-content-server-part:
-          name: Content Server Functional Test Part 1
+          name: Functional Test - Content 1
           index: 0
           total: 5
           requires:
             - Build
       - test-content-server-part:
-          name: Content Server Functional Test Part 2
+          name: Functional Test - Content 2
           index: 1
           total: 5
           requires:
             - Build
       - test-content-server-part:
-          name: Content Server Functional Test Part 3
+          name: Functional Test - Content 3
           index: 2
           total: 5
           requires:
             - Build
       - test-content-server-part:
-          name: Content Server Functional Test Part 4
+          name: Functional Test - Content 4
           index: 3
           total: 5
           requires:
             - Build
       - test-content-server-part:
-          name: Content Server Functional Test Part 5
+          name: Functional Test - Content 5
           index: 4
           total: 5
           requires:
             - Build
-
+      - playwright-functional-tests:
+          name: Functional Tests - Playwright
+          requires:
+            - Build
       - deploy-packages:
-          executor: docker-build-executor
+          name: FXA
+          requires:
+            - Lint
+            - Unit Test
+            - Integration Test 1
+            - Integration Test 2
+            - Integration Test 3
+            - Functional Test - Content 1
+            - Functional Test - Content 2
+            - Functional Test - Content 3
+            - Functional Test - Content 4
+            - Functional Test - Content 5
+            - Functional Tests - Playwright
+
+  nightly:
+    # This work flow runs a full build, test suite, and deployment of docker images nightly
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
           filters:
             branches:
-              ignore: /.*/
+              only: main
+    jobs:
+      - build:
+          name: Build (nightly)
```

Comment:
Adding a `(nightly)` suffix because of a quirk in circle ci where jobs with the same name would get a `-1` suffix sometimes. According to support forums this is intentional, but it creates some kind of awkward behaviors when using requires. This suffix guarantees each job has a unique name. This same sort of thing can be observed in other places in the config file.